docs/source/pages/DM6010.rst

```diff
@@ -0,0 +1,44 @@
+Y-adapter
+=========
+
+.. image:: /_static/images/DM6010/Y-adapter_front.png
+
+Overview
+********
+
+Y-adapter is used to **supply power to OAK cameras** through USB-C connector. Note that in most cases, host computer is
+able to provide enough current (<1A) and you don't need a Y-adapter. **Y-adapter is only needed** in cases where host is not able
```

Comment:
```suggestion
able to provide enough current (>1A) and you don't need a Y-adapter. **Y-adapter is only needed** in cases where host is not able
```

Reply:
Not sure what was meant here exactly but I think the intention was to use "greater" sign